src/components/shared/StackSettings.js

```diff
@@ -159,7 +157,7 @@ class StackSettingsImpl extends PureComponent<Props> {
               'cpp'
             )}
           </li>
-          {hasAllocations && !disableCallTreeSummaryButtons ? (
+          {hasAllocations ? (
```

Comment:
Thought: Ah, I think my original intent was to not show memory information in the stack chart.